### PR TITLE
REP-3454 Repose Lint Improvements

### PR DESCRIPTION
**Pull request by dmnjohns**
- Added a top-level foyer status
- Added a description alongside each foyer status
- Added a DEB/RPM start script

### CODE REVIEW

File: repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala [hunk 1]
```diff
@@ -487,6 +507,23 @@ object VerifyTryItNowCommand extends Command {
...
+      case FoyerStatus.NotAllowed =>
+        "Users with the 'foyer' Identity role WILL NOT pass through this component"
+      case FoyerStatus.Unknown =>
+        "Users with the 'foyer' Identity role MAY OR MAY NOT pass through this component"
+    }
+  }
+
   object FoyerStatus extends Enumeration {
     type FoyerStatus = Value
     val Allowed, AllowedAuthorized, AllowedNotAuthorized, NotAllowed, Unknown = Value
```
Comment (wdschei) [inline repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala]: `AllowedNotAuthorized` should have probably been `AllowedNoAuthorization`, but the comments and accompanying descriptions spell it out clear enough.

**Comment by wdschei** [comment]
Did you show @jorgew the new output?

File: repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala (hunk 1 above)
Comment (dmnjohns) [inline repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala]: I'm really bad at names.

I like AllowedNoAuthorization better, so I'll make the change. In which case I may also change "AllowedAuthorized" to "AllowedIfAuthorized". Unless you have a better suggestion for that one?

**Comment by jennyvo** [comment]
Let me know if you want to generate some outputs

File: repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala (hunk 1 above)
Comment (wdschei) [inline repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala]: That sounds good to me. I like `AllowedIfAuthorized` better too.

Comment (wdschei) [inline repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala]: Were you going to update these two names or are we just going to leave it to the descriptions?

**Comment by wdschei** [comment]
I think this all looks good. I like the warm fuzzy no-arg version and the oh crap verbose version.

File: repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala (hunk 1 above)
Comment (dmnjohns) [inline repose-aggregator/components/repose-lint/src/main/scala/org/openrepose/lint/commands/VerifyTryItNowCommand.scala]: Derp. I just 100% blanked on that. Doing it now.

**Comment by wdschei** [comment]
I like it!!!

**Pull request merged**